clean_code_main/clean_code_checks/y_check_base.clas.xml

```diff
@@ -12,56 +12,6 @@
     <UNICODE>X</UNICODE>
     <WITH_UNIT_TESTS>X</WITH_UNIT_TESTS>
    </VSEOCLASS>
-   <TPOOL>
-    <item>
-     <ID>I</ID>
-     <KEY>106</KEY>
-     <ENTRY>Maintain Attributes for the Code Inspector Check!</ENTRY>
-     <LENGTH>132</LENGTH>
-    </item>
-    <item>
-     <ID>I</ID>
-     <KEY>200</KEY>
-     <ENTRY>Consider Objects created after</ENTRY>
-     <LENGTH>60</LENGTH>
-    </item>
-    <item>
-     <ID>I</ID>
-     <KEY>201</KEY>
-     <ENTRY>Message Severity</ENTRY>
-     <LENGTH>26</LENGTH>
-    </item>
-    <item>
-     <ID>I</ID>
-     <KEY>202</KEY>
-     <ENTRY>Apply on Testcode</ENTRY>
-     <LENGTH>27</LENGTH>
-    </item>
-    <item>
-     <ID>I</ID>
-     <KEY>203</KEY>
-     <ENTRY>Threshold</ENTRY>
-     <LENGTH>19</LENGTH>
-    </item>
-    <item>
-     <ID>I</ID>
-     <KEY>204</KEY>
-     <ENTRY>Apply on Productive Code</ENTRY>
-     <LENGTH>48</LENGTH>
-    </item>
-    <item>
-     <ID>I</ID>
-     <KEY>300</KEY>
-     <ENTRY>Choose the Type of Code to be checked</ENTRY>
-     <LENGTH>60</LENGTH>
-    </item>
-    <item>
-     <ID>I</ID>
-     <KEY>301</KEY>
-     <ENTRY>Choose a Message Severity</ENTRY>
-     <LENGTH>50</LENGTH>
-    </item>
-   </TPOOL>
```

Comment:
why did you remove the text element?

Reply:
it looks like you removed the text elements from 35 classes

Reply:
if you change 38 Files to release one check we cannot do that.
the base class was changed, too. (please don't do that if you just want to release one check)

Reply:
??? I did not touched other classes than my newly created one !!! My workflow have been:
- Fork the original repository into a new repository
- Download the DEV branch to one of my development systems using the ABAP GIT client
- Add the new class to the same package
- Stage the package
- Create a Pull request
In that process, I did not changed any of the other classes. So this seems to be an issue of the GITclient and my development system....

Reply:
ok, now it looks better than before.